DEVELOPMENT.md

```
@@ -1,224 +1,140 @@
-* `RELATED_IMAGE_EBPF_AGENT`
-* `RELATED_IMAGE_FLOWLOGS_PIPELINE`
-* `RELATED_IMAGE_CONSOLE_PLUGIN`
-
-Examples:
+To link with unmerged changes, add this at the bottom of `go.mod`:
 
-```bash
-oc -n netobserv set env deployment/netobserv-controller-manager -c "manager" RELATED_IMAGE_EBPF_AGENT="quay.io/netobserv/netobserv-ebpf-agent:main"
-oc -n netobserv set env deployment/netobserv-controller-manager -c "manager" RELATED_IMAGE_FLOWLOGS_PIPELINE="quay.io/netobserv/flowlogs-pipeline:main"
-oc -n netobserv set env deployment/netobserv-controller-manager -c "manager" RELATED_IMAGE_CONSOLE_PLUGIN="quay.io/netobserv/network-observability-console-plugin:main"
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)